### ISSUE TITLE
Please consider marking this library as deprecated

### ISSUE THREAD

**Issue by mbifulco** [adamzapasnik/prettier-plugin-erb#137]
Hi there, 

I just had a _very_ bad time adding this library to a project I'm working on, and having it corrupt several files due to using the latest stable version of prettier. This package is unfortunately quite broken when used with prettier@latest, as explained #44.

This library hasn't been updated in a while, and I'd wager that it is causing more harm than good at this point - please consider using [npm's deprecation features](https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions) to stop this library from being added to new projects.

Thank you for your work on building and maintaining this project - I'm sure it was extremely valuable at one point. There comes a time to sunset projects, and this one seems to be at EOL unless it is going to be updated and maintained.

**Comment by brodyreid** [comment]
Yes, or could we please get an upgrade for compatibility for Prettier v3?

**Comment by yann120** [comment]
is there an alternative ?

**Comment by andyw8** [comment]
> is there an alternative ?

https://github.com/nebulab/erb-formatter

(not based on prettier)
